.devproxy/api-specs/sharepoint.yaml

```diff
@@ -57,32 +61,55 @@ paths:
     post:
       security:
         - delegated:
-          - AllSites.Read
-          - AllSites.Write
-          - AllSites.Manage
-          - AllSites.FullControl
+            - AllSites.Read
```

Comment:
I don't think it's necessary here to add the extra indentation. could you fix the indentation back for all changed lines? Just use 2 spaces to indent from the parent level